src/components/KnowledgeGraph/KnowledgeGraph.tsx

```diff
@@ -122,7 +217,17 @@ const KnowledgeGraph: FC<KnowledgeGraphProps> = ({
           zoom={0.3}
         />
       )}
-      <span>{selectedPaperTitle}</span>
+      <Button /// <reference path="buttonRef" />
+        size="large"
+        type="primary"
+        onClick={() => zoomOut()}
+      >
+        Zoom out
+      </Button>
+      <PaperInfo>
+        <h1>Paper Title:</h1>
+        <p>{selectedPaperTitle}</p>
+      </PaperInfo>
```

Comment:
Copy all this code to TestGraphPage.tsx and use code in src\components\TestTool\TestTool.tsx here.
Just add the "Send to KG" button to this page in addition to <SidebarError> component

Reply:
I meant to include the SidebarError to the component in next sprint tasks, because error handling in Knowledge graph would include more logic.

Reply:
Like what?

Reply:
Since it could be component is not loading due to the paperId not being available, but also due to an error when creating the component, where the user can only help in the first case.